templates/expo-template-bare-minimum/ios/Podfile

```diff
@@ -16,17 +16,21 @@ prepare_react_native_project!
 target 'HelloWorld' do
   use_expo_modules!
 
-  config_command = [
-    'node',
-    '--no-warnings',
-    '--eval',
-    'require(require.resolve(\'expo-modules-autolinking\', { paths: [require.resolve(\'expo/package.json\')] }))(process.argv.slice(1))',
-    'react-native-config',
-    '--json',
-    '--platform',
-    'ios'
-  ]
-  config = use_native_modules!(config_command)
+  if ENV['EXPO_USE_COMMUNITY_AUTOLINKING'] == '1'
+    config = use_native_modules!
+  else
+    config_command = [
+      'node',
+      '--no-warnings',
+      '--eval',
+      'require(require.resolve(\'expo-modules-autolinking\', { paths: [require.resolve(\'expo/package.json\')] }))(process.argv.slice(1))',
+      'react-native-config',
+      '--json',
+      '--platform',
+      'ios'
+    ]
+    config = use_native_modules!(config_command)
```

Comment:
the double `use_native_modules!` will introduce regression for #28705 again. we should find a way to deal with the problem

Reply:
considering people would need to explicitly install rnc-cli, maybe we can consider document how to do that or even add a separate config-plugin to support `EXPO_USE_COMMUNITY_AUTOLINKING`

Reply:
Yeah, I documented this here https://github.com/expo/expo/pull/32164

Reply:
> the double use_native_modules! will introduce regression for https://github.com/expo/expo/issues/28705 again. we should find a way to deal with the problem

this is the main problem that may break user and that's why i recommend not to have `EXPO_USE_COMMUNITY_AUTOLINKING` in our template but using config-plugin or manual change instead

Reply:
@gabrieldonadel - can we try something like `config = use_native_modules!(ENV['EXPO_USE_COMMUNITY_AUTOLINKING'] === '1' ? nil : config_command)` instead? maybe this reduces the risk of breaking config plugins with multiple `use_native_modules` occurrences

Reply:
hmm but would this be a problem even now that we're bumping from one SDK version to another? Wouldn't it be better if these dangerous config plugins weren't based on `use_native_modules!`? cc: @brentvatne

Reply:
yeah it would be but if we can accomplish the same goal but with less chance of it breaking config plugins (eg: the line i suggested above would work with our react-native-maps plugin) then i think we should go for the least breaking approach

Reply:
@brentvatne doing `config = use_native_modules!(ENV['EXPO_USE_COMMUNITY_AUTOLINKING'] === '1' ? nil : config_command)` is not really possible, in ruby we don't have an equivalent of the JS `undefined` for us to provide to this function and when we pass null it will error out stating 
```
[!] Expected a list_native_modules! to be called with a config command
    - Unable to autolink if no config is provided for the current project.

```

Reply:
updated it here https://github.com/expo/expo/pull/32258